src/parser/shaman/restoration/modules/talents/AncestralProtectionTotem.tsx

```diff
@@ -56,11 +61,12 @@ class AncestralProtectionTotem extends Analyzer {
         OR
         (type='${EventType.Cast}' AND ability.id=${SPELLS.TOTEMIC_REVIVAL_CAST.id})
       )`,
+      timeout: 2000,
```

Comment:
It feels wrong that WclOptions requires a timeout when it has a default value. Obviously not in the scope of this